### DISCUSSION TITLE
Hyperlink Version to Release Notes

### DISCUSSION THREAD

**Discussion by rbswift** [teslamate-org/teslamate#2367] [Ideas]
Could the version number in the car_live summary be linked to a release notes page like https://www.notateslaapp.com/software-updates/version/2021.44.30.11/release-notes ?

**Comment by cwanja** [comment]
Gave it a shot in #2390 PR. See what happens.

**Reply by rbswift** [reply]
looks great @cwanja thanks!

**Reply by cwanja** [reply]
~Should be readily available in 1.26.1~

Confirmed working in 1.26.1
